src/main/resources/db/changelog/20220328090000-accommodate_large_overrides.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202203288090000" author="JoshAlbrecht23">
+        <comment>
+            Accommodate large content overrides for consumers.
+        </comment>
+
+        <modifyDataType
+            columnName="value"
+            newDataType="clob"
```

Comment:
For stuff like this, we have a set of data types in datatypes.xml for Liquibase. In the case of unlimited-length strings, we have `text.type` which would be apt here.

Reply:
I see. Due to other changes, I changed the newDataType to varchar(1024)